config/application.yml.default

```diff
@@ -62,10 +62,12 @@ component_previews_enabled: false
 compromised_password_randomizer_threshold: 900
 compromised_password_randomizer_value: 1000
 country_phone_number_overrides: '{}'
+database_advisory_locks_enabled: false
 database_host: ''
 database_name: ''
 database_password: ''
 database_pool_idp: 5
+database_prepared_statements_enabled: false
```

Comment:
This looks like a good default, but do we need to get this propagated out to prod/staging/int/etc?  I looked in the prod application.yml file, and database_prepared_statements_enabled is not in there.